src/app/welcome/welcomedesk.component.html

```diff
@@ -0,0 +1,72 @@
+<style>
+  div.welcomedesk {
+      position: absolute;
+      top: 64px;
+      left: 50px;
+      padding-bottom: 100px;
+  }
+  div.welcomedesk div {
+      float: left;
+      margin: 20px 20px 30px 0;
+      width: 29%;
+      height: 100px;
+  }
+  div.welcomedesk h4 {
+      margin-bottom: 15px;
+  }
+  div.welcomedesk h4 mat-icon {
+      font-size: 1.4em;
+      height: 1em;
+      margin-right: 10px;
+  }
+</style>
+
+<div class="welcomedesk">
+  <h1>Welcome to Runbox</h1>
+  <p>Let's get you set up with Runbox 7 and make you an email pro!</p> 
+  <div>
+    <h4><mat-icon>network_check</mat-icon>Get up to speed</h4>
+    Get <a href="https://help.runbox.com/runbox-7-webmail-new-features/#Synchronized_account_index" target="help">synchronized</a> using the Runbox 7 app on all your devices.
+  </div>
+  
+  <div>
+    <h4><mat-icon>mail</mat-icon>Import your email</h4>
+    <a href="/mail/fetch" target="rmm6">Consolidate email</a> from your other email accounts.
+  </div>
+  
+  <div>
+    <h4><mat-icon>people</mat-icon>Import your contacts</h4>
+    Import all your <a href="/contacts">contacts</a> from other email services.
```

Comment:
Perhaps this should be a `routerLink`, so that it doesn't trigger a full page reload?

Reply:
Done in 4292cf1.